content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/While/1. While.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 1. While
+taxonomie:
+  - ib-19.2.Controlestructuur-Herhaling-While.OI
+  - ib-19.3.Controlestructuur-Herhaling-While.OI
+draft: false
+---
+
+## Wat is een While-loop?
+In de programmeerwereld worden **While-loops** gebruikt om een bepaald stukje code herhaaldelijk uit te voeren zolang aan een voorwaarde wordt voldaan. Dit maakt het mogelijk om dynamische herhalingen te creëren, waarbij je vooraf niet weet hoe vaak de loop zal draaien. De herhaling stopt zodra de voorwaarde **onwaar** (false) wordt.
+
+Zonder While-loops zou je telkens dezelfde code handmatig moeten herhalen, wat onpraktisch en foutgevoelig is. Met een While-loop kan een programma zichzelf herhalen en aanpassen op basis van variabele waarden en voorwaarden.
+
+> [!TIP] Casus
+> Stel je bedenkt een geheim getal, bijvoorbeeld 7. Iemand moet raden wat het getal is. Zolang het geraden getal niet klopt, vraag je opnieuw om een gok. Zodra de persoon het juiste getal raadt, stop je.
+
+## Hoe zit een While-loop in elkaar?
+De While-loop volgt een specifieke opbouw die begint met een While-clausule waarin een conditie wordt gespecificeerd. Zolang deze conditie waar (`true`) is, wordt de code binnen de lus herhaaldelijk uitgevoerd. Zodra de conditie onwaar (`false`) wordt, stopt de herhaling en gaat het programma verder.
```

Comment:
Suggestie: "...wordt gespecificeerd... 'binnen de ( )' "?
... Hangt natuurlijk wel van je gebruikte programmeertaal af.

Suggestie: "... de code 'binnen de accolades ( { } )' ". Inderdaad wel mooi om erbij te vermelden dat je dat deel de 'loop' of 'lus' noemt.